src/modules/0.0.17/aws_account/index.ts

```diff
@@ -1,11 +1,31 @@
 import { AWS, } from '../../../services/aws_macros'
 import { AwsAccountEntity, } from './entity'
-import { Context, Crud2, Mapper2, ModuleBase, } from '../../interfaces'
+import { Context, Crud2, MapperBase, ModuleBase, } from '../../interfaces'
 import * as metadata from './module.json' // TODO: Eliminate this?
 
+class AccountMapper extends MapperBase<AwsAccountEntity> {
+  module: AwsAccount;
+  entity = AwsAccountEntity;
+  equals = (_a: AwsAccountEntity, _b: AwsAccountEntity) => true;
```

Comment:
this is not related to this change, but why doesn't this have a comparison function?

Reply:
There's no equivalent record in the cloud, and the cloud mapper is a no-op that does nothing but return whatever is in the database on read and executes nothing on any of the others.

This module is not the best one to see what a "normal" module would look like in the new form, that's what the S3 module rewrite is for. ;)